static/app/components/events/eventErrors.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from '@emotion/styled';
+import isEqual from 'lodash/isEqual';
+import uniqWith from 'lodash/uniqWith';
+
+import {Alert} from 'sentry/components/alert';
+import {Error, ErrorItem} from 'sentry/components/events/errorItem';
+import List from 'sentry/components/list';
+import {JavascriptProcessingErrors} from 'sentry/constants/eventErrors';
+import {t, tn} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Artifact, Project} from 'sentry/types';
+import {Event} from 'sentry/types/event';
+import {defined} from 'sentry/utils';
+import {useQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import {DataSection} from './styles';
+
+const MAX_ERRORS = 100;
+
+type EventErrorsProps = {
+  event: Event;
+  proGuardErrors: Array<Error>;
+  projectSlug: Project['slug'];
+};
+
+const getURLPathname = (url: string) => {
+  try {
+    return new URL(url).pathname;
+  } catch {
+    return undefined;
+  }
+};
+
+export const EventErrors = ({event, proGuardErrors, projectSlug}: EventErrorsProps) => {
+  const organization = useOrganization();
+  const orgSlug = organization.slug;
+
+  const releaseVersion = event.release?.version;
+  const sourceCodeErrors = (event.errors ?? []).filter(
+    error => error.type === 'js_no_source' && error.data.url
+  );
+
+  const pathNames: Array<string> = [];
+
+  for (const sourceCodeError of sourceCodeErrors) {
+    const url = sourceCodeError.data.url;
+    if (url) {
```

Comment:
probably don't need to check for url again since you've filtered by it on line 41